week2-6.3-long_cat_is_long.py

```diff
@@ -0,0 +1,25 @@
+import re
+
+
+def count_words(text):
+    """
+    this function receives a string, uses a generator expression to remove the all special characters
+    and store all the words in a list, then uses a dictionary comprehension to loop through that list
+    and store the name and the length of each word and returns it to be printed, the program imports
+    regular expression module to extract words only from the text.
+    :param text: a string to operate on
+    :return: a dictionary that has all the words from the text as keys and the values are the length of each word
+    """
+    clean = (word for word in re.split("\W+", text) if word.isalpha())
+    result = {word.lower(): len(word) for word in clean}
+
+    return result
+
+
+print(count_words("""
```

Comment:
Should be inside a function.
Also, the string should be kept in a variable or a constant (preferably) and not being defined inside the function calling.